src/borg/platform/linux.pyx

```diff
@@ -240,19 +241,7 @@ cdef _sync_file_range(fd, offset, length, flags):
 cdef unsigned PAGE_MASK = sysconf(_SC_PAGESIZE) - 1
 
 
-def _is_WSL():
-    """detect Windows Subsystem for Linux"""
-    try:
-        with open('/proc/version') as fd:
-            linux_version = fd.read()
-        # hopefully no non-WSL Linux will ever mention 'Microsoft' in the kernel version:
-        return 'Microsoft' in linux_version
-    except:  # noqa
-        # make sure to never ever crash due to this check.
-        return False
-
-
-if _is_WSL():
+if 'basesyncfile' in workarounds:
     class SyncFile(BaseSyncFile):
         # if we are on Microsoft's "Windows Subsytem for Linux", use the
```

Comment:
According to the description in `usage_general.rst.inc` from above, this no longer affects WSL only.

Reply:
yup, I'll make that more general.

Reply:
done.